packages/docs/page-config/navigationRoutes.ts

```diff
@@ -267,9 +267,16 @@ export const navigationRoutes: NavigationRoute[] = [
         name: "color-input",
         displayName: "Color Input",
       },
-
+      
       {
         category: 'Layout',
+        name: 'layout',
```

Comment:
We agreed to add a couple of examples at the beginning, similarly to https://www.naiveui.com/en-US/os-theme/components/layout

It seems to me that ready layout is something the user would hope to bring back from docs.